src/components/Settings/Settings.js

```diff
@@ -1,13 +1,25 @@
 import React from 'react';
-import { func, string } from 'prop-types';
+import { func, string, object, arrayOf } from 'prop-types';
 
+import { noop } from 'redux-saga/utils';
```

Comment:
:neckbeard: Super minor (non-blocking), and not something our linting is currently configured to catch -- in all other files, our imports are organized as:

1. group of by absolute package imports
2. whitespace
3. grouped by relative app imports

In that vein, this line should go above the whitespace rather than below.